templates/kube-scheduler.yaml

```diff
@@ -32,6 +32,35 @@ spec:
     volumeMounts:
     - name: kube-scheduler-policy-config
       mountPath: {{kube_scheduler_policy_config_dir}}
```

Comment:
'if' block should be closed here. Config directory should be mounted in the scheduler container only when kube_scheduler_policy_config_dir is set.

Reply:
Fixed this